gensim/models/__init__.py

```diff
@@ -13,7 +13,7 @@
 from .logentropy_model import LogEntropyModel  # noqa:F401
 from .word2vec import Word2Vec  # noqa:F401
 from .doc2vec import Doc2Vec  # noqa:F401
-from .keyedvectors import KeyedVectors  # noqa:F401
+from .word2vec import Word2VecKeyedVectors as KeyedVectors  # noqa:F401
```

Comment:
An old variant is better (like from @jayantj) with `KeyedVectors = SomeNewClass` by documentation reasons (KeyedVectors does not exist as a class -> never be in the documentation).